mona_sdk/client.py

```diff
@@ -541,8 +541,13 @@ def _create_client_response(
         if total > 0 and not response.ok:
             try:
                 result_info = response.json()
-                failed = result_info["failed"]
-                failure_reasons = result_info["failure_reasons"]
+                top_level_error = result_info.get("topLevelError")
```

Comment:
please add comment that there are two methods (maybe some context - which/why etc) and probably todo to canonize...